Add version 8 message to master README (#4655)
TerriaJS/terriajs: README.md

TerriaJS is a library for building rich, web-based geospatial data explorers, used to drive [National Map](http://nationalmap.gov.au), [AREMI](http://nationalmap.gov.au/renewables) and [NEII Viewer](http://neii.gov.au/viewer/).  It uses [Cesium](https://cesiumjs.org) and WebGL for a full 3D globe in the browser with no plugins.  It gracefully falls back to 2D with [Leaflet](http://leafletjs.com/) on systems that can't run Cesium. It can handle catalogs of thousands of layers, with dozens of geospatial file and web service types supported. It is almost entirely JavaScript in the browser, meaning it can even be deployed as a static website, making it simple and cheap to host.

--------------------

**We are currently working on a brand new version of Terria &mdash; verson 8!**

We will soon be posting docs outlining this new version: 

> - how it differes from the current version 

Reviewer: ```suggestion
- how it differs from the current version 
```

- how it will impact users (including guides and tools to help you upgrade)

> - release roadmap (including which features are supported, and which we are in development)

Reviewer: ```suggestion
- release roadmap (including which features are supported or in development)
```
